tests/onnxruntime/test_modeling.py

```diff
@@ -2742,6 +2747,188 @@ def test_compare_generation_to_io_binding(self, test_name: str, model_arch: str,
         gc.collect()
 
 
+class ORTModelForVision2SeqIntegrationTest(ORTModelTestMixin):
+    # TODO: speech_to_text should be tested
+    SUPPORTED_ARCHITECTURES = ["vision-encoder-decoder", "trocr"]
+
+    FULL_GRID = {
+        "model_arch": SUPPORTED_ARCHITECTURES,
+        "use_cache": [False, True],
+    }
+
+    ORTMODEL_CLASS = ORTModelForVision2Seq
+
+    TASK = "vision2seq-lm"
+
+    def exclude_trocr_with_cache(params):
+        if params[0] == "trocr" and params[1] == True:
+            return None
+        return params
```

Comment:
Just for my knowledge, why is this not supported? What will happen if an user tries to use `ORTModelForVision2Seq` with `trocr` with `use_cache = True`?

Reply:
The model currently not output any `past_key_values` when `use_cache=True`.  So need to figure out why. Probably something wrong in the modeling code.

```
| What will happen if an user tries to use ORTModelForVision2Seq with trocr with use_cache = True?
```

For this I have added an error message during export.